xpdan/startup/save_server.py

```diff
@@ -35,6 +35,7 @@ def setup_saver(doc, **kwargs):
     "{start[sample_name]}_"
     # The writers handle the trailing ``_``
     "{human_timestamp}_"
+    "{sp_computed_exposure}s_"
```

Comment:
This should pull from the start document `{start[...]}`. It could pull from the descriptor though if that information is in the descriptor